src/components/pages/NotFound/PageNotFound.jsx

```diff
@@ -1,27 +1,25 @@
 import React from "react";
 import { Helmet } from "react-helmet-async";
 import "./PageNotFound.scss";
-import Logo from "../../../utils/rct-logo.png";
 import { useNavigate } from "react-router-dom";
+import strings from "../../../config/strings";
+
 const PageNotFound = () => {
   const navigate = useNavigate();
   return (
     <>
       <Helmet>
         <title>404 Page not found</title>
       </Helmet>
-      <div className="header">
-        <img src={Logo} className="logoImage" alt="Website logo" />
-      </div>
       <div className="pageNotFoundContent">
-        <h1>404 - Page not found.</h1>
-        <h2>Unfortunately, this page doesn't exist.</h2>
+        <h1>404 - Not Found</h1>
+        <h1>Se pare ca pagina nu a fost gasita.</h1>
         <h2
           onClick={() => {
             navigate("/home");
```

Comment:
Can we also have these paths somewhere global?

Reply:
Created a ticket: https://radiocampustransilvania.atlassian.net/browse/W3RCT-43